src/entities/catalogs/services/get-pageviews-by-catalog-id.ts

```
@@ -0,0 +1,76 @@
+import { BetaAnalyticsDataClient, protos } from "@google-analytics/data";
+
+const analyticsDataClient = new BetaAnalyticsDataClient({
+  credentials: {
+    client_email: process.env.GOOGLE_CLIENT_EMAIL,
+    // TODO: Consider storing private key as encoded base64, then decode and use
+    private_key: process.env
+      .GOOGLE_ANALYTICS_PRIVATE_KEY!.split(String.raw`\n`)
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)